docs/vocabulary.md

```diff
@@ -337,7 +361,8 @@ pipeline in corresponding documentation.
 
 | Project            |          |          |          |          |         |              |
 |--------------------|----------|----------|----------|----------|---------|--------------|
-| **CircleCI**       | N/A      | Step     | Job      | Workflow | Trigger | Executor     |
+| **ArgoCD**         | Sync Wave         | Sync Phase  | Sync      |N/A      | Event         |  Application Controller            |
+| **CircleCI**       | N/A      | Step     | Job      | Workflow | Trigger | Executor     |                    |          |          |          |          |         |              |
```

Comment:
Looks like there's some extra |s here that was accidently added?

Reply:
Thanks for catching that - indeed accidental (IDE introduced?)